ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/sflow_settings.schema.yml

```diff
@@ -15,6 +15,13 @@ keys:
       The sFlow process will only be configured if any interface is enabled for sFlow.
       For default enabling of sFlow for various interface types across the fabric see `fabric_sflow`.
     keys:
+      sample:
```

Comment:
```suggestion
      sample_rate:
```

Reply:
There are few more keys inside sample, so I have restructured the data-model now.